substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/c/libc/MuslLibc.java

```diff
@@ -64,24 +64,22 @@ public void prepare(Path directory) {
         setUpSpecFile(directory);
     }
 
+    @Override
+    public List<String> getAdditionalQueryCodeCompilerOptions() {
+        List<String> options = new ArrayList<>();
+        /* Avoid the dependency to muslc for builds cross compiling to muslc. */
+        options.add("--static");
+        return options;
```

Comment:
Arrays.asList("--static")

Reply:
I agree, the code is also cleaner that way. I have changed it to `Collections.asSingletonList`.